superdesk/templates/translation_changed.html

```diff
@@ -1,5 +1,6 @@
 {% extends "email_layout.html" %}
 
 {% block content %}
-<p>{{username}} changed the following translated article: <a href="{{link}}">{{link}}</a></p>
+<p>The following original item has been modified by {{username}}.
```

Comment:
I think it can use
```
...</p>
<p>...
```
so it will start on new line. line break doesn't do much in html